Language/Variables/Implicitly_Induced_Getters_and_Setters/definition_A05_t01.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Case ⟨Setter: Mutable variable with no declared type, no
+/// initialization⟩. A variable declaration of the form static? late? var id;
+/// implicitly induces a setter with the header void set id(dynamic x), whose
+/// execution sets the value of id to the incoming argument x.
+///
+/// @description Checks that the static type of an implicit getter of a mutable
+/// variable with no declared type and no initialization is `dynamic`
+/// @author sgrekhov22@gmail.com
+
+late var x1;
+var x2;
+
+class C {
+  static late var v1;
+  late var v2;
+  var v3;
+  static var v4;
+}
+
+main() {
+  x1 = -1;
```

Comment:
Let's just use `Expect.throws(() => x1.whatever)` as usual.

Reply:
Again. Changed description to `setter`. `Expect.throws(() => x1.whatever)` will test getter, not setter